x-pack/test_serverless/functional/test_suites/search/search_index_detail.ts

```diff
@@ -54,12 +54,14 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await pageObjects.svlSearchIndexDetailPage.expectConnectionDetails();
       });
 
-      it.skip('should show api key', async () => {
-        await pageObjects.svlApiKeys.deleteAPIKeys();
-        await svlSearchNavigation.navigateToIndexDetailPage(indexName);
-        await pageObjects.svlApiKeys.expectAPIKeyAvailable();
-        const apiKey = await pageObjects.svlApiKeys.getAPIKeyFromUI();
-        await pageObjects.svlSearchIndexDetailPage.expectAPIKeyToBeVisibleInCodeBlock(apiKey);
+      describe('API key details', () => {
```

Comment:
this is ok, but I'd actually like to see us more these and the start page API key tests into a different file so that we can tag them to skip if they continue to be flakey.

Reply:
Moving the API key tests looks unnecessary because we’d need to create a test for API keys on every page. We already have a flow where we check API keys on the current page. The current setup, with tests in describe block lets us skip flaky tests if needed
https://github.com/elastic/kibana/blob/e03e59b6d482a05435d86a612d92028f264df893/x-pack/test_serverless/functional/test_suites/search/elasticsearch_start.ts#L95